lib/Avatar/styles.scss

```diff
@@ -153,7 +152,9 @@ $avatar-pillbox-padding: $typo-size-lead/4 $typo-size-lead/3 !default;
     text-overflow: ellipsis;
     white-space: nowrap;
   }
+}
 
+.avatar__wrapper--additional-visible {
```

Comment:
Always a grey area in BEM but maybe `avatar__wrapper--additional-visible` could use a state class instead e.g. `is-visible` or `is-open`. 

The reason for this is 1 disadvantage of BEM is that you can stick too much to the modifier principle. Making it less legible. This is how I see it;

**Use a modifier** 

- when a components take a new state (e.g. component--highlighted)

**Use a state class**

- when a components element or modifier has altered due to a user interaction
- examples: `is-open`, `is-active`, `is-visible`

There isn't no right answer (as always) but seeing as you are adding to an existing modifier I recommend using a state class 😄

Reply:
You can see this principle in the same component...`.is-active .avatar__caret` works better than `avatar__caret--active` 😄